docs/en/02_Developer_Guides/01_Templates/01_Syntax.md

```diff
@@ -293,7 +287,7 @@ Methods can also be chained.
 Inside the loop scope, there are many variables at your disposal to determine the current position in the list and 
 iteration.
 
- * `$Even`, `$Odd`: Returns boolean, handy for zebra striping.
+ * `$Even`, `$Odd`: Returns boolean. Handy for zebra striping.
```

Comment:
@willmorgan Are you suggesting we add $Pos for striping with more than 2 colors ? $Pos is already in list.